src/Requests.jl

```diff
@@ -28,7 +29,7 @@
 
     function default_request(method,resource,host,data,user_headers=Dict{None,None}())
         headers = @compat Dict(
-            "User-Agent" => "Requests.jl/0.0.0",
+            "User-Agent" => "Requests.jl/"*string(Pkg.installed("Requests")),
```

Comment:
`Requests.jl` verison should be a constant, you do not want to calculate it every time you make a request.